jupyter_server/utils.py

```diff
@@ -26,8 +26,12 @@ def isawaitable(f):
     class ConcurrentFuture:
         """If concurrent.futures isn't importable, nothing will be a c.f.Future"""
         pass
+    from urllib.parse import quote, unquote, urlparse, urljoin
+    from urllib.request import pathname2url
+except ImportError:
+    from urllib import quote, unquote, pathname2url
+    from urlparse import urlparse, urljoin
```

Comment:
This block didn't cherry-pick correctly.  Just take the first pair of lines and left-justify.  The last three are py2 related - so can be deleted.